### DISCUSSION TITLE
install frigate+ , only need the api in config?

### DISCUSSION THREAD

**Discussion by kiloptero** [blakeblackshear/frigate#9339] [Ask A Question]
hi all, there is any else you need to start using more object that just put the api in the config part "plus_api_key"?

**Comment by kiloptero** [comment]
model:
  path: plus://e63b7345cc83a84ed79dedfc99c16616

i see this in a frigate.yaml..
need to change the e63b7345cc83a84ed79dedfc99c16616 for my api key?

**Reply by NickM-27** [reply]
you need to add your api key and then after generating your first model you will be given a model ID and that is what you will put there

**Comment by kiloptero** [comment]
thanks, but only to use the addtional object (like face of ups).. need
something else,??

Andres Roepke Del Solar.
Ingeniero Civil Industrial
Ingeniero Electronico
Mobile 88196932

**Reply by NickM-27** [reply]
You'll need to add the objects to the object track list

**Comment by kiloptero** [comment]
ok! thanks

Andres Roepke Del Solar.
Ingeniero Civil Industrial
Ingeniero Electronico
Mobile 88196932
